src/main/java/io/vertx/redis/client/impl/CommandReporter.java

```diff
@@ -11,8 +11,8 @@
 
 import java.util.function.Function;
 
-class CommandReporter {
-  enum Tags {
+public class CommandReporter {
+  public enum Tags {
```

Comment:
This feels wrong, and this class is in the `impl` package, so it shouldn't be relied upon anyway.